src/planner.ts

```diff
@@ -61,24 +61,63 @@ export function geometryToGeoJSON(
   };
 }
 
+const entranceQuery = `
+[out:json][timeout:25];
+(
+  relation(around:10, LAT, LON)[building];
+  way(r);
+  way(around:10, LAT, LON)[building];
+)->.b;
+// gather results
+(
+  node(w.b)[entrance];
+);
+// print results
+out body;
+>;
+out skel qt;
+`;
+
 export default function calculatePlan(
   origin: [number, number],
   destination: [number, number],
   callback: (f: FeatureCollection) => void
 ): void {
-  const planner = new FlexibleTransitPlanner();
-  planner
-    .query({
-      from: { latitude: origin[0], longitude: origin[1] },
-      to: { latitude: destination[0], longitude: destination[1] },
-      roadNetworkOnly: true
+  const url = new URL("https://overpass-api.de/api/interpreter");
+  url.searchParams.append(
+    "data",
+    entranceQuery
+      .replace(/LAT/g, destination[0].toString())
+      .replace(/LON/g, destination[1].toString())
+  );
+  fetch(url.toString()).then(response =>
+    response.json().then(body => {
```

Comment:
Try existing types or this one from app.quicktype.io:
```ts
export interface OverpassResponse {
    version:   number;
    generator: string;
    osm3s:     Osm3S;
    elements:  Element[];
}

export interface Element {
    type: string;
    id:   number;
    lat:  number;
    lon:  number;
    tags: Tags;
}

export interface Tags {
    entrance:  string;
    operator?: string;
    ref?:      string;
}

export interface Osm3S {
    timestamp_osm_base: Date;
    copyright:          string;
}
```

Reply:
Implemented.